internal/namespaces/instance/v1/custom_image.go

```diff
@@ -93,3 +94,28 @@ func imageListBuilder(c *core.Command) *core.Command {
 
 	return c
 }
+
+// imageCreateBuilder overrides 'instance image create' to rename the argument 'root-volume' into 'snapshot-id'.
+func imageCreateBuilder(c *core.Command) *core.Command {
+	type CreateImageRequestCustom struct {
+		*instance.CreateImageRequest
+		SnapshotID string
+	}
+
+	c.ArgSpecs.GetByName("root-volume").Name = "snapshot-id"
+
+	c.ArgsType = reflect.TypeOf(CreateImageRequestCustom{})
+
+	c.Run = func(ctx context.Context, args interface{}) (i interface{}, e error) {
+		requestCustom := args.(*CreateImageRequestCustom)
+
+		request := requestCustom.CreateImageRequest
+		request.RootVolume = requestCustom.SnapshotID
+
+		client := core.ExtractClient(ctx)
```

Comment:
Can you use the old run instead of these 3 lines? It's better for maintainability.